pkg/manager/member/utils.go

```diff
@@ -182,6 +182,7 @@ func serviceEqual(new, old *corev1.Service) (bool, error) {
 // setUpgradePartition set statefulSet's rolling update partition
 func setUpgradePartition(set *apps.StatefulSet, upgradeOrdinal int32) {
 	set.Spec.UpdateStrategy.RollingUpdate = &apps.RollingUpdateStatefulSetStrategy{Partition: &upgradeOrdinal}
+	glog.Infof("set %s/%s partition to %d successfully", set.GetNamespace(), set.GetName(), upgradeOrdinal)
```

Comment:
I think it's not a good idea to log in the utility function. log when the API operation has done?

Reply:
removed the `successfully ` word. ptal